src/github.rs

```diff
@@ -1240,6 +1262,29 @@ impl IssuesEvent {
 #[derive(Debug, serde::Deserialize)]
 struct PullRequestEventFields {}
 
+#[derive(Debug, serde::Deserialize)]
+pub struct WorkflowRunJob {
+    pub id: u64,
+    pub run_id: u64,
+    pub name: String,
+    pub workflow_name: String,
```

Comment:
Currently we only need the SHA, right? So we can just avoid deserializing all the rest, and don't have to define the conclusion.

Reply:
Yes, we currently only need the SHA. I have ideas for the other fields but we can add them later. Only left `head_sha`.